### ISSUE TITLE
engine "node" is incompatible with this module

### ISSUE THREAD

**Issue by rasgo-cc** [webpack-contrib/terser-webpack-plugin#340]
- Operating System: Win10
- Node Version: 12.10
- NPM Version: 6.10.3
- webpack Version: 4.41.2
- terser-webpack-plugin Version: 5.3.6 (`latest` - tried with 4.x.x and same result)

### Expected Behavior

Should install.

### Actual Behavior

```
error terser@5.3.6: The engine "node" is incompatible with this module. Expected version "^10.0.0,^11.0.0,^12.0.0,>=14.0.0". Got "12.10.0"
error Found incompatible module
```

Current node version (12.10.0) is actually compatible with semver "^12.0.0", but reported the opposite.

**Comment by evilebottnawi** [comment]
No `5.3.6` version https://github.com/webpack-contrib/terser-webpack-plugin/blob/master/package.json#L3

**Comment by evilebottnawi** [comment]
https://github.com/terser/terser/issues/856

**Comment by evilebottnawi** [comment]
Sorry we can't fix it on our side, wait upstream

**Comment by evilebottnawi** [comment]
@ccambournac update your deps

**Comment by ccambournac** [comment]
Done, but having the following Webpack error with `terser-webpack-plugin 5.0.0`. 

```
TypeError: Cannot read property 'javascript' of undefined
    at compiler.hooks.compilation.tap.compilation (/node_modules/terser-webpack-plugin/dist/index.js:366:38)
    ...
```

Should I open an issue?

**Comment by ccambournac** [comment]
Arg, updated too fast. My repo does not support Webpack 5 yet.

**Comment by evilebottnawi** [comment]
Yes, latest, version supports only webpack@5, you can use previously version if you use `webpack@4`

**Comment by ccambournac** [comment]
I rolled back to 4.2.3 version and it is ok now with updated terser dependency pointing to fixed 5.3.7 version.

Thanks @evilebottnawi!
